session/session_test.go

```diff
@@ -3237,6 +3237,185 @@ func (s *testSessionSuite2) TestPerStmtTaskID(c *C) {
 	c.Assert(taskID1 != taskID2, IsTrue)
 }
 
+func (s *testSessionSuite3) TestSetVarHint(c *C) {
```

Comment:
```
[2020-10-14T16:52:47.899Z] ----------------------------------------------------------------------

[2020-10-14T16:52:47.899Z] FAIL: session_test.go:3240: testSessionSuite3.TestSetVarHint

[2020-10-14T16:52:47.899Z] 

[2020-10-14T16:52:47.899Z] session_test.go:3354:

[2020-10-14T16:52:47.899Z]     tk.MustQuery("SELECT /*+ SET_VAR(default_tmp_storage_engine='CSV') */ @@default_tmp_storage_engine;").Check(testkit.Rows("1"))

[2020-10-14T16:52:47.899Z] /home/jenkins/agent/workspace/tidb_ghpr_check_2/go/src/github.com/pingcap/tidb/util/testkit/testkit.go:63:

[2020-10-14T16:52:47.899Z]     res.c.Assert(resBuff.String(), check.Equals, needBuff.String(), res.comment)

[2020-10-14T16:52:47.899Z] ... obtained string = "[CSV]\n"

[2020-10-14T16:52:47.899Z] ... expected string = "[1]\n"

[2020-10-14T16:52:47.899Z] ... sql:SELECT /*+ SET_VAR(default_tmp_storage_engine='CSV') */ @@default_tmp_storage_engine;, args:[]

[2020-10-14T16:52:47.899Z] 

[2020-10-14T16:52:47.899Z] PASS: session_test.go:537: testSessionSuite.TestTxnLazyInitialize	0.043s

[2020-10-14T16:52:47.899Z] PASS: session_test.go:1932: testSessionSuite2.TestInformationSchemaCreateTime	1.028s

[2020-10-14T16:52:47.899Z] PASS: session_test.go:1855: testSessionSuite2.TestIssue986	0.009s

[2020-10-14T16:52:47.899Z] 

[2020-10-14T16:52:47.899Z] ----------------------------------------------------------------------
```

seems this test is not stable?

Reply:
This is my spelling error, it has been fixed.